rs/bitcoin/kyt/src/providers.rs

```diff
@@ -0,0 +1,72 @@
+use bitcoin::Network;
+use ic_btc_interface::Txid;
+use ic_cdk::api::management_canister::http_request::{
+    CanisterHttpRequestArgument, HttpHeader, HttpMethod, TransformContext, TransformFunc,
+};
+
+pub fn create_request(
+    network: Network,
+    txid: Txid,
+    max_response_bytes: u32,
+) -> CanisterHttpRequestArgument {
+    match network {
+        Network::Bitcoin => btcscan_request(txid, max_response_bytes),
+        Network::Testnet => mempool_space_testnet_request(txid, max_response_bytes),
+        _ => panic!("{} network is not supported", network),
```

Comment:
But `regtest` is defined as a `network` variant. Isn't this inconsistent?

Reply:
Obviously, things would have to work differently for `regtest`. We will probably never support. Is there a reason to keep it as a variant?

Reply:
Think I'll just remove it to avoid the confusion.